pkgs/stdenv/darwin/make-bootstrap-tools.nix

```diff
@@ -37,8 +37,7 @@ in rec {
   build = stdenv.mkDerivation {
     name = "stdenv-bootstrap-tools";
 
-    nativeBuildInputs = [ buildPackages.nukeReferences buildPackages.cpio ]
-      ++ lib.optionals targetPlatform.isAarch64 [ buildPackages.darwin.sigtool ];
+    nativeBuildInputs = [ buildPackages.nukeReferences ];
```

Comment:
```suggestion
    nativeBuildInputs = [ nukeReferences ];
```

Reply:
I'm guessing this may have been done for a reason (weird splicing stuff), as it was present in the before diff.

Reply:
Hmm, it doesn't make sense to me either. I removed it.

It builds fine without it, though we're no longer cross-compiling, so any package set will work now, I think.